augur/filter.py

```diff
@@ -376,16 +376,27 @@ def run(args):
     if args.subsample_seed:
         random.seed(args.subsample_seed)
     num_excluded_subsamp = 0
-    if args.group_by and (args.sequences_per_group or args.subsample_max_sequences):
+    #if args.group_by and (args.sequences_per_group or args.subsample_max_sequences):
```

Comment:
The new conditional statement looks good, so you can delete the commented version.